docs/install/quickstart-install.md

```diff
@@ -6,12 +6,19 @@ components:
 function: tutorial
 ---
 
-# Install Knative using quickstart
+# Install the Knative quickstart plugin
```

Comment:
This topic covers both installing the plugin, and using the plugin to get a cluster with Knative installed on it.

Maybe just "quickstart" or "quickstart install"?